sentry/tests/test_client.py

```diff
@@ -6,12 +6,16 @@
 import unittest
 
 import raven
+from mock import patch
```

Comment:
As per pre-commit (please rebase):

```suggestion
from unittest.mock import patch
```